secant/Resources/Assets.xcassets/WelcomeScreenLogo.imageset/Contents.json

```diff
@@ -12,12 +12,12 @@
           "value" : "dark"
         }
       ],
-      "filename" : "DarkLogo.png",
+      "filename" : "ecc-black 3.png",
```

Comment:
Shouldn't be here logo? Or we want ecc-black here? Could we please name files on disk little bit better? In same way as previous logo. Use Dark/Light prefix and @3/2/whatever suffixes.